open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/janus-connector/pom.xml

```diff
@@ -67,6 +67,19 @@
             <artifactId>repository-services-apis</artifactId>
         </dependency>
 
+
+        <!-- berkeley & lucene aren't compile dependencies, so need exception for maven dependency plugin -->
+        <dependency>
```

Comment:
are these the only supported options, or can a user configure with a different db/indexer? If so, I wonder if they should be hard dependencies... or rather should be provided as part of the install/environment setup - in this case they would not be dependencies at all, but it would be the responsibility of the deployer to ensure janus is configured.

it somewhat depends if you have a 'hidden, embedded; graph, or one you expect to be configured... since an enterprise is probably use cassandra, I think perhaps they shouldn;'t be a dependency>?

Reply:
Hi Nigel, 

These dependencies should indeed by managed dynamically, I think Mandy and Graham are working on the architectural design to make this possible?

Reply:
If we need to make it a compile dependency now, perhaps one approach would be to open up an issue to track removing that hard dependency. like 'make janus berkeley/lucene loading dynamic' or similar? At least then we'll have an issue to follow ?